monkey/common/credentials/credentials.py

```diff
@@ -2,13 +2,34 @@
 
 from typing import Optional, Union
 
+from pydantic import SecretBytes, SecretStr
+
 from ..base_models import InfectionMonkeyBaseModel
 from . import LMHash, NTHash, Password, SSHKeypair, Username
 
 Secret = Union[Password, LMHash, NTHash, SSHKeypair]
 Identity = Username
 
 
+def get_plain_text(secret: Union[SecretStr, SecretBytes, None, str]) -> Optional[str]:
```

Comment:
We use "plaintext" (one word) everywhere else in the code.
```suggestion
def get_plaintext(secret: Union[SecretStr, SecretBytes, None, str]) -> Optional[str]:
```

Reply:
In fact, maybe something like this would be more explanatory:
```suggestion
def get_plaintext_secret(secret: Union[SecretStr, SecretBytes, None, str]) -> Optional[str]:
```